modules/gatsby/src/components/site-query.jsx

```diff
@@ -41,6 +41,7 @@ const QUERY = graphql`
         ADDITIONAL_LINKS {
           name
           href
+          index
```

Comment:
If you are adding features, please document them!

Reply:
Done